cli/command/task.go

```diff
@@ -10,6 +14,157 @@ func NewTaskCommand(b *CommandBase) *TaskCommand {
 	return &TaskCommand{b}
 }
 
-func (e *TaskCommand) Command() cli.Command {
-	return cli.Command{}
+func (t *TaskCommand) Command() cli.Command {
+	return cli.Command{
+		Name:  "task",
+		Usage: "manage layer0 tasks",
+		Subcommands: []cli.Command{
+			{
+				Name:      "create",
+				Usage:     "create a new task",
+				Action:    t.create,
+				ArgsUsage: "ENVIRONMENT NAME DEPLOY",
+				Flags: []cli.Flag{
+					cli.IntFlag{
+						Name:  "copies",
+						Value: 1,
+						Usage: "number of copies of deploy to run (default: 1)",
```

Comment:
I don't think we need `(default:1)` here since the cli package we use prints out the default for you. I could be wrong though

Reply:
[nit] remove `(default: 1)`